ddtrace/_trace/utils_botocore/span_tags.py

```diff
@@ -14,6 +15,7 @@
 from ddtrace.internal.constants import COMPONENT
 from ddtrace.internal.utils.formats import deep_getattr
 
+_PAYLOAD_TAGGER=AWSPayloadTagging()
```

Comment:
nit: Missing a space.  The linter is also failing.

Hint: you can run `hatch -v run lint:fmt` to auto-format your files (check hatch.toml for other commands)